.github/ISSUE_TEMPLATE.md

```diff
@@ -13,6 +13,10 @@ provide the basic information we require.
 
 -->
 
+Please look thru your error log for the string `gyp info using node-gyp@` and if the version number is less than the [current release of node-gyp](https://github.com/nodejs/node-gyp/releases) then __please upgrade__ using the instructions at https://github.com/nodejs/node-gyp/blob/master/docs/Updating-npm-bundled-node-gyp.md and try your command again.
+
+[`node-sass is deprecated`](https://github.com/nodejs/node-gyp/issues?q=label%3A%22Node+Sass+--%3E+Dart+Sass%22) so we can not provide support for it here.
```

Comment:
this sounds like we could provide support for things that _aren't_ deprecated, we have to be careful, maybe:

```suggestion
Requests for help with [`node-sass` are very common](https://github.com/nodejs/node-gyp/issues?q=label%3A%22Node+Sass+--%3E+Dart+Sass%22). Please be aware that this package is deprecated, you should seek alternatives and avoid opening new issues about it here.
```